docs/guides/high-availability/multi-cluster/deploy-keycloak-kubernetes.adoc

```diff
@@ -0,0 +1,54 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Deploying {project_name} for HA with the Operator"
+summary="Deploy {project_name} for high availability with the {project_name} Operator as a building block."
+tileVisible="false" >
+
+This guide describes advanced {project_name} configurations for {kubernetes} which are load tested and will recover from single Pod failures.
+
+These instructions are intended for use with the setup described in the <@links.ha id="multi-cluster-concepts"/> {section}.
+Use it together with the other building blocks outlined in the <@links.ha id="multi-cluster-building-blocks"/> {section}.
+
+== Prerequisites
+
+* {kubernetes} cluster running.
+* Understanding of a <@links.operator id="basic-deployment" /> of {project_name} with the {project_name} Operator.
+* AWS Aurora database deployed using the <@links.ha id="multi-cluster-deploy-aurora" /> {section}.
+* {jdgserver_name} server deployed using the <@links.ha id="multi-cluster-deploy-infinispan-kubernetes-crossdc" /> {section}.
+* Running {project_name} with OpenJDK 21, which is the default for the containers distributed for {project_name}, as this enabled virtual threads for the JGroups communication.
```

Comment:
There is not JGroups communication in multi-cluster. This line can be removed :)